desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraph.java

```diff
@@ -456,4 +458,28 @@ private void setScrollbarValue(JScrollBar scrollBar, int value) {
         scrollBar.setValues(value, scrollModel.getExtent(), scrollModel.getMinimum(), scrollModel.getMaximum());
     }
 
+    /**
+     * Checks if a job including its child jobs are correctly configured.
+     * 
+     * @return
+     */
+    private boolean checkAnalysisJobBuilderIsConfigured() {
+
+        boolean isConfigurated = true;
+        final List<AnalyzerComponentBuilder<?>> analyzerComponentBuilders = _analysisJobBuilder
+                .getAnalyzerComponentBuilders();
+        for (AnalyzerComponentBuilder<?> analyzerComponentBuilder : analyzerComponentBuilders) {
+            final List<OutputDataStream> outputDataStreams = analyzerComponentBuilder.getOutputDataStreams();
+            if (!outputDataStreams.isEmpty()) {
+                for (OutputDataStream outputDataStream : outputDataStreams) {
+                    final AnalysisJobBuilder outputDataStreamJobBuilder = analyzerComponentBuilder
+                            .getOutputDataStreamJobBuilder(outputDataStream);
```

Comment:
First check if the output data stream is being consumed - or else you will create new job builders on the fly with this call, which is not what we want.